javascript/inavapp.js

```diff
@@ -1,10 +1,7 @@
-(function() {
```

Comment:
Just for the sake of getting it functioning, the auto-executing javascript closure isn't needed here.